### PR TITLE
Patched minesweeper

### PR DESCRIPTION
**Pull request by NattCaat**
- Uncovered zero fields are no longer shown as exploded mines
- Increased timeout from 1 minute to 3 minutes
- Small changes in documentation

### CODE REVIEW

File: DemonOverlord/config/cmd_info.json [hunk 1]
```diff
@@ -174,7 +174,7 @@
                 "command": "minesweeper",
                 "short": "ms",
                 "syntax": "minesweeper {action}",
-                "description": "start a game of minesweeper",
+                "description": "Play a game of minesweeper\nTo start a game, use 'minesweeper start'\nFurther instruction are given during game",
```
Comment (Dragonsight91) [inline DemonOverlord/config/cmd_info.json:177]: maybe give the user a hint in both the game embed and the command description, that there's a 3 min time window for inputs. That way users can't complain about not knowing the time they have

File: DemonOverlord/core/modules/minesweeper.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def msg_test(message: discord.Message) -> bool:
         while running and not game_won:
             try:
                 message = await command.bot.wait_for(
-                    "message", check=msg_test, timeout=60
+                    "message", check=msg_test, timeout=180
```
Comment (Dragonsight91) [inline DemonOverlord/core/modules/minesweeper.py:59]: maybe use this value and plonk it into the embed somehow...

**Review by Dragonsight91 — CHANGES_REQUESTED** [review]
mostly good, few suggestions, see comments

**Comment by NattCaat** [comment]
Player gets notified that there is a time window for inputs in game embed. I have not figured out yet, how to integrate it in the help embed

**Review by Dragonsight91 — APPROVED** [review]
Seems fine to me it has my approval
will be merged now

**Pull request merged**